lib/hello.js

```diff
@@ -5,7 +5,11 @@ var cors = require('cors');
 function helloRoute() {
   var hello = new express.Router();
   hello.use(cors());
-  hello.use(bodyParser());
+  //hello.use(bodyParser());
```

Comment:
Yes this gets rid of the deprecate warnings, this was a part of Corinne's changes. I can remove this and not update body-parser version?